features/lorawan/lorastack/phy/LoRaPHY.cpp

```diff
@@ -1343,7 +1300,7 @@ lorawan_status_t LoRaPHY::add_channel(const channel_params_t *new_channel,
     // Default channels don't accept all values
     if (id < phy_params.default_channel_cnt) {
         // Validate the datarate range for min: must be DR_0
-        if (new_channel->dr_range.fields.min > phy_params.min_tx_datarate) {
+        if (new_channel->dr_range.fields.min != DR_0) {
```

Comment:
Should this be something like this:
        // Validate the datarate range for min: must be greater than min_tx_datarate
        if (new_channel->dr_range.fields.min > phy_params.min_tx_datarate) {
            dr_invalid = true;
        }

Reply:
@kivaisan Actually no. This code is correct because for default channels minimum data-rate is 0 always. So if it is not DR_0, it's invalid.

Reply:
Ok.